Framework/Algorithms/src/CreatePSDBleedMask.cpp

```diff
@@ -217,26 +217,20 @@ bool CreatePSDBleedMask::performBleedTest(
   const size_t topEnd(midIndex - numIgnoredPixels / 2);
   const size_t bottomBegin(midIndex + numIgnoredPixels / 2);
 
-  /// Is the input a distribution or raw counts. If true then bin width division
-  /// is necessary when calculating the rate
-  bool isRawCounts = !(inputWS->isDistribution());
-
   const int numBins = static_cast<int>(inputWS->blocksize());
   std::vector<double> totalRate(numBins, 0.0);
   size_t top = 0, bot = bottomBegin;
   for (; top < topEnd; ++top, ++bot) {
     const int topIndex = tubeIndices[top];
     const int botIndex = tubeIndices[bot];
-    auto &topY = inputWS->y(topIndex);
-    auto &botY = inputWS->y(botIndex);
+    // Provide frequencies as opposed to calculating frequencies on the fly for
+    // count daya
+    auto &topY = inputWS->frequencies(topIndex);
+    auto &botY = inputWS->frequencies(botIndex);
```

Comment:
No, as discussed yesterday you cannot do these changes as long as `ConvertToDistribution` and `WorkspaceHelpers::makeDistribution` exist!